Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -171,11 +171,13 @@ void WebFullScreenManager::enterFullScreenForElement(WebCore::Element* element)
 
     setElement(*element);
 
-    bool isVideoElement = false;
+    bool isVideoElementWithControls = false;
 #if PLATFORM(IOS_FAMILY) || (PLATFORM(MAC) && ENABLE(VIDEO_PRESENTATION_MODE))
+    if (auto* videoElement = dynamicDowncast<HTMLVideoElement>(element))
```

Comment:
what about `HTMLAudioElement`?  should we use `HTMLMediaElement` instead (since that covers both)?